src/odb/src/defin/definSNet.cpp

```diff
@@ -516,6 +537,9 @@ void definSNet::pathViaArray(const char* via_name,
                                     numY,
                                     stepX,
                                     stepY,
+                                    0,
+                                    0,
+                                    0,
```

Comment:
Any time you have constants like these it's good to append the param name as a block comment.

```c++
/*x=*/ 0,
/*y=*/ 0,
/*z=*/ 0,

```